setup.cfg

```diff
@@ -15,12 +15,10 @@ classifiers=
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Programming Language :: Python
-    Programming Language :: Python :: 2.7
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.5
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
```

Comment:
There is also a "Python 3 only" classifier, just in case.

`Programming Language :: Python :: 3 :: Only`